mscore/pianoroll/pianolevelschooser.cpp

```diff
@@ -24,24 +25,55 @@ PianoLevelsChooser::PianoLevelsChooser(QWidget *parent)
 
       connect(levelsCombo, SIGNAL(activated(int)), SLOT(setLevelsIndex(int)));
       connect(setEventsBn, SIGNAL(clicked(bool)), SLOT(setEventDataPressed()));
-}
+      }
 
+//---------------------------------------------------------
+//   setPianoView
+//---------------------------------------------------------
+
+void PianoLevelsChooser::setPianoView(PianoView* pianoView)
+      {
+      _pianoView = pianoView;
+      }
 
 //---------------------------------------------------------
-//   PianoLevelsChooser
+//   updateSetboxValue
+//---------------------------------------------------------
+
+void PianoLevelsChooser::updateSetboxValue()
```

Comment:
Is it really necessary in the PR where you just improve tooltips?

Reply:
Edward J Sanchez asked me to add a feature where the the Set box would be updated if the user clicked on a note.

Reply:
I've changed the write up for this issue.  It's now no longer about tooltips.